pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -80,7 +80,7 @@ func (r *REST) NewList() runtime.Object {
 var _ = rest.Creater(&REST{})
 
 var (
-	ForbiddenNames    = []string{"openshift", "kubernetes", "kube"}
```

Comment:
We reserved entire patterns and these are the roots of those patterns.

What is a CRO and why is it messing with a list for the project request endpoints?

Reply:
CRO = cluster resource override; a cluster admin feature for driving overcommit on a cluster by adjusting pod resource requests/limits at admission time.

https://docs.openshift.com/container-platform/3.7/admin_guide/overcommit.html

I used the forbidden names as an exemption list here.  Logic was that if the project is reserved, we probably don't want CRO enforced on it as well. https://github.com/openshift/origin/pull/16845

There is an annotation to exempt the project from CRO, but we didn't want to include it on all present and feature reserved projects.  Most people are not aware of CRO in the Online clusters and it causes issues for infrastructure services like logging, metrics, registry, and router.